src/js/player.js

```diff
@@ -2941,92 +2958,145 @@ Player.prototype.options_ = {
+   * with respect to rendering the current playback position, from the
+   * codes in the list below.
+   * - HAVE_NOTHING (numeric value 0)
+   *   No information regarding the media resource is available.
+   * - HAVE_METADATA (numeric value 1)
+   *   Enough of the resource has been obtained that the duration of the
+   *   resource is available.
+   * - HAVE_CURRENT_DATA (numeric value 2)
+   *   Data for the immediate current playback position is available.
+   * - HAVE_FUTURE_DATA (numeric value 3)
+   *   Data for the immediate current playback position is available, as
+   *   well as enough data for the user agent to advance the current
+   *   playback position in the direction of playback.
+   * - HAVE_ENOUGH_DATA (numeric value 4)
+   *   The user agent estimates that enough data is available for
+   *   playback to proceed uninterrupted.
+   *
+   * @see https://html.spec.whatwg.org/multipage/embedded-content.html#dom-media-readystate
+   * @return {Number} the current playback rendering state
+   * @method Player.prototype.readyState
+   */
+  'readyState'
+].forEach((fn) => {
+  Player.prototype[fn] = function() {
+    return this.techGet_(fn);
+  };
+});
 
+/* document methods *?
 /**
  * Fired when the user agent begins looking for media data
  *
  * @event loadstart
+ * Player.prototype.handleTechLoadStart_
  */
-Player.prototype.handleTechLoadStart_; // eslint-disable-line
 
 /**
  * Fired when the player has initial duration and dimension information
  *
  * @event loadedmetadata
+ * Player.prototype.handleLoadedMetaData_
  */
-Player.prototype.handleLoadedMetaData_; // eslint-disable-line
 
 /**
  * Fired when the player receives text data
  *
  * @event textdata
+ * @method Player.prototype.handleTextData_
  */
-Player.prototype.handleTextData_; // eslint-disable-line
 
 /**
  * Fired when the player has downloaded data at the current playback position
  *
  * @event loadeddata
+ * @method Player.prototype.handleLoadedData_
  */
-Player.prototype.handleLoadedData_; // eslint-disable-line
 
 /**
  * Fired when the user is active, e.g. moves the mouse over the player
  *
  * @event useractive
+ * @method Player.prototype.handleUserActive_
  */
-Player.prototype.handleUserActive_; // eslint-disable-line
 
 /**
  * Fired when the user is inactive, e.g. a short delay after the last mouse move or control interaction
  *
  * @event userinactive
+ * @method Player.prototype.handleUserInactive_
  */
-Player.prototype.handleUserInactive_; // eslint-disable-line
 
 /**
  * Fired when the current playback position has changed *
  * During playback this is fired every 15-250 milliseconds, depending on the
  * playback technology in use.
  *
  * @event timeupdate
+ * @method Player.prototype.handleTimeUpdate_
  */
-Player.prototype.handleTimeUpdate_; // eslint-disable-line
-
-/**
- * Fired when video playback ends
- *
- * @event ended
- */
-Player.prototype.handleTechEnded_; // eslint-disable-line
```

Comment:
We are but I think this is the same or similar to: #3630 as @event just attached the event to the global scope in the docs. I can put `@triggers` on the method if we want though